grpc/core/src/main/java/io/helidon/grpc/core/MarshallerSupplier.java

```diff
@@ -71,7 +71,10 @@ public <T> MethodDescriptor.Marshaller<T> get(Class<T> clazz) {
             if (MessageLite.class.isAssignableFrom(clazz)) {
                 return proto.get(clazz);
             }
-            return JavaMarshaller.instance();
+            String msg = String.format(
+                    "Attempting to use class %s, a Java serialization marshaller is no longer supported by default",
+                    clazz.getName());
+            throw new IllegalArgumentException(msg);
```

Comment:
Throwing exception here makes sense, but I wouldn't even mention `JavaMarshaller` in the error message.

Assuming that `proto` is now the default marshaller (which it pretty much is), I would change the message to:

"Class %s must be a valid ProtoBuf message, or a custom marshaller for it must be specified explicitly."

I think that is much more appropriate, as it provides a hint to user on what to do.